UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -721,18 +750,17 @@ public void StopExternalBleeding()
 		/// </summary>
 		public void InternalBleedingLogic(AttackType attackType = AttackType.Internal, DamageType damageType = DamageType.Brute)
 		{
-			float damageToTake = UnityEngine.Random.Range(MinMaxInternalBleedingValues.x, MinMaxInternalBleedingValues.y);
 			if(currentInternalBleedingDamage >= maximumInternalBleedDamage)
 			{
-				BodyPart currentParent = ContainedIn;
+				BodyPart currentParent = GetParent();
```

Comment:
this is here because you'll be never accessing the root body part controller since its internal bleeding, you would never be internally on the outermost body part

Reply:
I have read this multiple times and I'm not following, what exactly is the problem here?

Reply:
well, you would never have internal bleeding on a limb? 
Outer most Body part, so therefore you would never get root body part as a parent, 
also this code here causes errors when it does